boards/sodaq-sara-aff/include/gpio_params.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2018 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_sodaq-sara-aff
+ * @{
+ *
+ * @file
+ * @brief       Board specific configuration of direct mapped GPIOs
+ *
+ * @author      Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
+ */
+
+#ifndef GPIO_PARAMS_H
+#define GPIO_PARAMS_H
+
+#include "board.h"
+#include "saul/periph.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief    GPIO pin configuration
+ */
+static const  saul_gpio_params_t saul_gpio_params[] =
+{
+    {
+        .name = "LED Builtin",
+        .pin  = LED0_PIN,
+        .mode = GPIO_OUT,
+    },
+    {
+        .name = "LED Red",
+        .pin = LED1_PIN,
+        .mode = GPIO_OUT
+    },
+    {
+        .name = "LED Green",
+        .pin = LED2_PIN,
+        .mode = GPIO_OUT
+    },
+    {
+        .name = "LED Blue",
+        .pin = LED3_PIN,
+        .mode = GPIO_OUT
+    }
```

Comment:
LED1-3 seem to be connected low-side and should get proper saul flags
(something like `.flags  = (SAUL_GPIO_INVERTED | SAUL_GPIO_INIT_CLEAR)`)